src/main/java/org/interledger/codecs/oer/OerIA5StringCodec.java

```diff
@@ -41,15 +39,15 @@ public OerIA5String read(
   }
 
   @Override
-  public void write(
-      final CodecContext context, final OerIA5String instance, final OutputStream outputStream
-  ) throws IOException {
+  public void write(final CodecContext context, final OerIA5String instance,
+      final OutputStream outputStream) throws IOException {
 
     Objects.requireNonNull(context);
     Objects.requireNonNull(instance);
     Objects.requireNonNull(outputStream);
 
-    final byte[] data = instance.getValue().getBytes();
+    final byte[] data = instance.getValue()
+        .getBytes();
```

Comment:
I'd rather have a universal standard that just works, even if it does horribly ugly things like that. I wonder if we can include a formatter in the project that the IDE's would use auto-magically?